doc/ot_api_doc.h

```diff
@@ -53,6 +53,7 @@
  * @defgroup api-net                  IPv6 Networking
  * @{
  *
+ * @defgroup api-ble-secure           BLE
```

Comment:
Let's use "BLE Secure" here since it's specifically for TLS over BLE, not just regular BLE